src/index.html

```diff
@@ -11,7 +11,172 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+
+        <label for="surname">Surname:</label>
+        <input
+          type="text"
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          class="form-input-gap"
+        >
```

Comment:
1) i'd recommend to wrap input with label to avoid using id/for
2) apply class to your wrapper (label) instead of input to make outer indent
```suggestion
        <label>
          Surname:
          <input
            type="text"
            name="surname"
            autocomplete="off"
            class="form-input-gap"
          >
        </label>
```